Add `Solo Score Rush` tournament article (#13861)
ppy/osu-wiki: wiki/Tournaments/SSR/1/en.md

8. Players may ask for a rematch if they encounter technical issues while playing within 30 seconds or 25% of the beatmap drain length (whichever happens first) of the game's start. Referees may, at their discretion, veto this request.
   - "Lag spikes" are not considered a valid reason to rematch a beatmap.
   - This rule is not to be abused. Referees may veto a rematch request if they find that this is the case.
9. If a player disconnects mid-game, their score will not be counted towards their cumulative, unless adequate proof of said score is provided. The following are considered as acceptable proof:
   - Player point-of-view live stream snippets (commonly referred to as "clips" or "VODs"). The entirety of the play, along with the results screen must be clearly visible along with the affected player's score.

>    - Replay files of the play, taken directly from the "Local scores" tab on the affected player's client (the timestamps must exactly match the time at which the game took place, as seen on the multiplayer lobby link).

Reviewer: ```suggestion
   - Replay files of the play, taken directly from the "local scores" tab on the affected player's client (the timestamps must exactly match the time at which the game took place, as seen on the multiplayer lobby link).
```

    - Breaking the chat rules may result in a silence. Silenced players cannot participate in multiplayer matches for the duration of the punishment and, as a result, can have their match forfeited.
14. Penalties for violating the tournament rules include, but are not limited to:
    - Removal of pick rights for a map or an entire match.
    - Point deductions.
    - Declaring the match as forfeited, or as a win by default for the other player.
    - Disqualification from the entire tournament.
    - Disqualification from the current and future official tournaments, until appealed.
15. Referees may allow, at their discretion, lower or higher tolerances for timers.
16. The tournament organisers may request liveplays or recordings of individual players at any point in the tournament without prior warning.
17. The tournament organisers will be responsible for receiving and investigating any tournament-related complaints.
18. The tournament organisers reserve the right to modify these rules at any moment. Any such changes will be announced in advance.

### Registration

> 1. Every user interested in playing will be be required to sign up through the Player Registrations form.

Reviewer: ```suggestion
1. Every user interested in playing will be be required to sign up through the player registrations form.
```

Also no big deal, but the link to the form is not present in the article. Kinda whatever as the tournament is, y'know, over. But just in case you want to add it under the "Links" section.

2. To ensure valid registrations, every qualifying participant will be manually checked through the Tournament screening process.
3. Participants will be briefed on further steps via an announcement message at the end of the Player Registrations form.
4. Tournament staff members are not allowed to play in the tournament, with the exception of streamers, commentators, and graphic designers. Eliminated players may be enlisted as mappool playtesters and/or replay creators. Note that per conventional Officially supported tournament rules, eliminated players may be enlisted as referees, however this role is not open for such individuals at the moment.

>    - The tournament managers must ensure that such staff members do not have access to priviledged information, e.g. Qualifiers score information, mappool information before the mappool showcase, etc. They are to be treated as regular players whenever possible.

Reviewer: More correct to add the example as an in-lined parenthesis + corrected "privileged"
```suggestion
   - The tournament managers must ensure that such staff members do not have access to privileged information (e.g. Qualifiers score information, mappool information before the mappool showcase). They are to be treated as regular players whenever possible.
```

   - 1st-32nd seeds: The top 32 seeds of the Qualifiers will advance directly into the Round of 64 of the tournament. They do not need to participate in any matches during Play-offs.
   - 33rd-96th seeds: Players within this group will participate in one match during Playoff Stage. Their opponent will be a player in the 33rd to 96th seed group — more specifically, (129 - the player's seed). Players who win their match in this stage will advance to Group Stage.

##### Bracket stage onwards

1. The Bracket Stage is a double elimination stage. Players advancing to this stage must lose two matches in this stage to be eliminated from the tournament.
2. After Play-offs, the remaining players will be seeded based on their performance in Play-offs. This will be used to determine the remaining players' opponents for the bracket stage. In particular, their opponent for the bracket stage will be (65 - the player's seed).
3. In a winners bracket match, the player who meets the winning condition in a match will be declared the winner of the match and will move further into the bracket. The loser of the match will fall into the losers bracket.

> 4. In a losers bracket match, the player who meets the winning condition in a match will be declared the winner of the match and will move further into the bracket. The loser of the match will be eliminated from the tournament.

Reviewer: ```suggestion
4. In a losers bracket match, the player who meets the winning condition will be declared the winner of the match and will move further into the bracket. The loser of the match will be eliminated from the tournament.
```

5. Players who are in the winners bracket are required to play in one match in each stage. The only exception to this is if the player is the sole player in winner bracket in the Grand Finals stage, where if the player were to lose their first match in the Grand Finals stage, they will be required to play in a second match.
6. Players who are in the losers bracket at the start of the Round of 16 or further are required to play in two matches in a single stage. However, players who lose their first match in the losers bracket in those stages will be eliminated from the tournament, and thus no longer need to play in any further matches.

##### Match protocol

1. Matches for this stage of the tournament will have players play an amount of picks (and depending on the match situation, a 'Final Scoring' map:
   - Each round has a maximum amount of maps to be picked.
   - For Play-offs, Round of 64, and Round of 32, that amount is 7, including the 'Final Scoring'.
   - For Round of 16 and Quarterfinals, that amount is 9, including the 'Final Scoring'.
   - For Semifinals, Finals, and Grand Finals, that amount is 11, including the 'Final Scoring'.

>    - If the match is x maps away from the maximum amount of maps, and the cumulative score gap of a player is higher than that of the other player by 50,000*x or more, the player with the higher cumulative score gap is declared the winner.

Reviewer: ```suggestion
   - If the match is `x` maps away from the maximum amount of maps, and the cumulative score gap of a player is higher than that of the other player by `50,000 * x` or more, the player with the higher cumulative score gap is declared the winner.
```

   - Grand Finals: Stage 1: 0ㅣStage 2: 0ㅣStage 3: 0ㅣStage 4: 0.08ㅣStage 5: 0.06ㅣStage 6: 0.41ㅣStage 7: 0.12ㅣStage 8: 0.33

### Match procedures

1. A referee will create a multiplayer room 10 minutes before the scheduled match time. Players must join the lobby in that period.
   - The room settings are Game mode: "osu!mania", Team mode: "Head-to-head", and Score mode: "ScoreV2". The room name must follow the pattern of SSR: ({Player 1}) vs. ({Player 2}). The player mentioned first in the room name must occupy the first space in the room's player list, the player mentioned second in the room name must occupy the second space in the room's player list.
2. Each player must use !roll once in the match's chatroom. After the rolls are set, match protocols will happen as described in BRACKET STAGE INFORMATION, Match Protocol, 3., where the player with the higher roll will be Player A.
3. Each player will receive one "tactical timeout" of 2 minutes, to be used as extra time to ban, pick, or ready for a beatmap. The tactical timeout may not be called on the opposing player's pick or ban phase.

> 4. Players will be allowed 90 seconds for each pick/ban phase (as described on the bracket stage instructions section), and an additional 90 seconds to press the Ready button on their client before play commences. If a player takes more time than allotted for either action, the procedures adopted will be as follows:

Reviewer: ```suggestion
4. Players will be allowed 90 seconds for each pick/ban phase (as described on the bracket stage instructions section), and an additional 90 seconds to press the 'Ready' button on their client before play commences. If a player takes more time than allotted for either action, the procedures adopted will be as follows:
```

   - On subsequent occurrences:
     - For a pick timer: a random map will be chosen from the mappool by rolling a random number, based on the amount of beatmaps that were neither picked nor banned.
     - For a ready timer: the referee will issue the start command on a 10-second countdown. In this scenario, if the referee starts the match and either player is not present in the lobby, their opponent will receive 25,000 added to their cumulative score gap.
     - Repeat offenders may receive further sanctions from the tournament managers.

### Scheduling information

1. Each stage will be held on a single weekend. In the event there are more than 96 qualifying players, Play-offs will be held across two weekends instead.
2. Matches in Qualifiers will be held during specific time slots proposed by the tournament managers. Each player may choose a time slot in which they want to play, or request an EX slot when no timeframes are suitable (EX slot requests may be rejected if there are insufficient referees for the time of the slot), at a time no later than July 20, 20:00 UTC.
   - A player that does not pick a time slot by July 20, 20:00 UTC will be considered disqualified.

> 3. All bracket stages are scheduled to be held between Saturday 00:00 and Sunday 23:59 UTC.

Author: I'll specify later.
